website/docs/commands/cli-config.html.markdown

```diff
@@ -52,18 +48,31 @@ disable_checkpoint = true
 
 The following settings can be set in the CLI configuration file:
 
-* `disable_checkpoint` - when set to `true`, disables
+- `disable_checkpoint` — when set to `true`, disables
   [upgrade and security bulletin checks](/docs/commands/index.html#upgrade-and-security-bulletin-checks)
   that require reaching out to HashiCorp-provided network services.
 
-* `disable_checkpoint_signature` - when set to `true`, allows the upgrade and
+- `disable_checkpoint_signature` — when set to `true`, allows the upgrade and
   security bulletin checks described above but disables the use of an anonymous
   id used to de-duplicate warning messages.
 
-* `plugin_cache_dir` - enables
+- `plugin_cache_dir` — enables
   [plugin caching](/docs/configuration/providers.html#provider-plugin-cache)
   and specifies, as a string, the location of the plugin cache directory.
 
+- `credentials` — provides credentials for use with Terraform Enterprise's
+    [private module registry.](/docs/enterprise-beta/registry/index.html)
+    This setting is a repeatable block, where the block label is a hostname
+    (either `atlas.hashicorp.com` or the hostname of your private install) and
```

Comment:
In earlier discussions we had planned to ship with the private registry at `app.terraform.io` rather than at `atlas.hashicorp.com`, so that we don't need to break all the existing references when we shut down `atlas.hashicorp.com`. Did that plan change?